container-images/mds-event-processor/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:8-alpine
+
+RUN apk add --no-cache tini postgresql-client
+
+RUN mkdir /mds
+
+COPY dist/* /mds/
+
+WORKDIR /mds
+
+ENTRYPOINT ["/sbin/tini", "node", "index.js", "--type=event"]
```

Comment:
I don't see where/how these `--type=` arguments are used in the code.

Reply:
deprecated with new handlers, need to remove